styles/pages/statistics.module.scss

```diff
@@ -0,0 +1,15 @@
+.stats {
+	display: flex;
+	flex-wrap: wrap;
+	padding-block: 3.5rem;
+	column-gap: 3.5rem;
```

Comment:
if the column and row gap are the same, use `gap` property instead

Reply:
noted